basic-examples/config-file/README.md

```diff
@@ -0,0 +1,73 @@
+# Using Config Files
+
+## Workflow Description
+
+This workflow runs a simple metadata collection step that uses Ansible's gather facts
+plugin, and returns only the system hostname. In order to collect the metadata in the
```

Comment:
Re "Ansible's gather facts plugin", the actual plugin appears to be named `arcaflow-plugin-metadata`...do we really want to mention "Ansible" here?  The _step_ is named `ansible_facts`, but there's no obvious reason why (and that doesn't mention "gather", either).

I recommend going generic:  here, reference the name of the Arca plugin (e.g., "Metadata"), and, in the workflow, call the step something like "get_host_info" or "get_host_env" or "metadata".

Reply:
The Arcaflow plugin uses Ansible, which itself has a plugin called gather facts. It's turtles all the way down.

Reply:
Perhaps so, but here above the crust, there is no visible sign of Ansible (other than a weird reference in the workflow which uses a plugin called "metadata").

In the documentation here, the fact that the _plugin_ uses Ansible is a useless detail, which it would be better to omit.  If you want to refer to the plugin, you should use Arca's name for it (as we do in the other workflow which uses it).